framework/encode/ags_dispatch_table.h

```diff
@@ -47,7 +53,8 @@ struct AgsDispatchTable
     AGS_DRIVEREXTENSIONSDX12_PUSHMARKER    agsDriverExtensionsDX12_PushMarker{ nullptr };
     AGS_DRIVEREXTENSIONSDX12_POPMARKER     agsDriverExtensionsDX12_PopMarker{ nullptr };
     AGS_DRIVEREXTENSIONSDX12_SETMARKER     agsDriverExtensionsDX12_SetMarker{ nullptr };
-    AGS_INIT_NOT_SUPPORTED                 agsInitNotSupported{ nullptr }; // In case, old version dll is used by the application
+    AGS_INIT_NOT_SUPPORTED agsInitNotSupported{ nullptr }; // In case, old version dll is used by the application
```

Comment:
Is the line be formatted?

Reply:
I believe clang formatted it like this in order to fit the comment on the same line.